src/handlers/int-test-support/helpers/envHelper.ts

```diff
@@ -8,3 +8,12 @@ export const configStackName = (): string => `di-btm-cfg-${configName()}`;
 
 export const runViaLambda = (): boolean =>
   process.env.TEST_VIA_LAMBDA === "true";
+
+export const outputQueue = (): string => {
+  if (
+    process.env.OUTPUT_QUEUE_URL === undefined ||
```

Comment:
I'd do this check at the beginning of the handler like in other lambdas.

Also this is a test-helper, so would be good to have another helper in actual production code for this.

Reply:
Oops, I should have checked the directory name 🫢